tests/unit/test_regioninfo.py

```diff
@@ -46,11 +46,7 @@ def setUp(self):
 
     def test_load_endpoint_json(self):
         endpoints = load_endpoint_json(boto.ENDPOINTS_PATH)
-        self.assertTrue('ec2' in endpoints)
-        self.assertEqual(
-            endpoints['ec2']['us-east-1'],
-            'ec2.us-east-1.amazonaws.com'
-        )
+        self.assertTrue('partitions' in endpoints)
```

Comment:
Nevermind ``load_endpoints_json`` is misleading. Did not realize that all it interface does is load a json file. There is nothing specific to endpoints in that function. The change should be fine.